README.md

```diff
@@ -24,9 +24,39 @@ $ CONFIG_ENV=stage-prod npm run dev
 
 In a nextjs project, you can call `loadConfiguration()` in `next.config.js` and pass the result to next as `env`, see https://nextjs.org/docs/api-reference/next.config.js/environment-variables - configuration values will be available on `process.env` both client- and server-side
 
-In any node process, simply call `loadConfiguration()` in an appropriate place and access variables on `process.env`
+```
+const loadConfiguration = require("@carforyou/configuration")
+const configuration = loadConfiguration()
+module.exports = {
+  env: configuration
+}
+```
+
+In any node process, simply call `loadConfiguration()` in your entry point and access variables on `process.env`.
+
+```
+require("@carforyou/configuration")
+loadConfiguration()
```

Comment:
I don't fully get that if you don't assign the default export of the module to a name like `const loadConfiguration = require("@carforyou/configuration")` how does that work in terms of exporting the named function?

Reply:
yes! this example is totally flawed, thanks!